### PR TITLE
epics do not need to be time estimated #138

### PR DESCRIPTION
**Pull request by naazy**
epics do not need to be time estimated #138

### CODE REVIEW

**Comment by iteles** [comment]
Adding @danwhy and @shouston3 as potential reviewers as they've shown interest in dwylbot in the past. If you guys have some spare time during the workday, please take a look 😊

File: web/controllers/rules/issue/time_estimation.ex [hunk 1]
```diff
@@ -20,10 +20,11 @@ defmodule Dwylbot.Rules.Issue.TimeEstimation do
 
     labels = payload["issue"]["labels"]
     in_progress = Enum.any?(labels, fn(l) -> l["name"] == "in-progress" end)
+    epic = Enum.any?(labels, fn(l) -> l["name"] == "epic" end)
     estimation = labels
     |> Enum.map(fn(l) -> Regex.match?(~r/T\d{1,3}[mhd]/, l["name"]) end)
     |> Enum.reduce(false, fn(x, acc) -> x || acc end)
-    if in_progress && !estimation do
+    if in_progress && !epic && !estimation do
```
Comment (samhstn) [inline web/controllers/rules/issue/time_estimation.ex]: If working with `boolean` values with `if` functions, perhaps consider using [`and` `or` and `not`](https://elixir-examples.github.io/examples/boolean-operators). They are stricter and fairly readable

```elixir
if in_progress and not epic and not estimation do
```

**Comment by naazy** [comment]
Have addressed your comments @shouston3. Thanks :smile:

**Review by samhstn — APPROVED** [review]
Great

**Pull request merged**